plugins/PostProcessingPlugin/scripts/TimeLapse.py

```diff
@@ -66,6 +66,22 @@ def getSettingDataString(self):
                     "type": "float",
                     "default_value": 9000,
                     "enabled": "park_print_head"
+                },
+                "retract":
+                {
+                    "label": "Retraction Distance",
+                    "description": "Filament retraction distance for camera trigger.",
+                    "unit": "mm",
+                    "type": "int",
+                    "default_value": 1
```

Comment:
```suggestion
                    "default_value": 0
```

In order to keep the script behaving the same if the user didn't change anything, this should be 0. We try to ensure that scripts like this stay the same.

Another option would be to create an upgrade script that checks if someone had the script loaded (and then set this value to 0), but that's a bit more complicated.